code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -1,9 +1,21 @@
+/mob/living/carbon/human/get_movespeed_modifiers()
+	var/list/considering = ..()
+	. = considering
+	if(has_trait(TRAIT_IGNORESLOWDOWN))
+		. = null
+		for(var/id in .)
+			var/list/data = .[id]
+			if(data[MOVESPEED_DATA_INDEX_FLAGS] & IGNORE_NOSLOW)
+				.[id] = data
+
+/mob/living/carbon/human/update_config_movespeed()
+	add_movespeed_modifier(MOVESPEED_ID_HUMAN_CONFIG_SPEEDMOD, FALSE, 100, override = TRUE, legacy_slowdown = get_config_multiplicative_speed())

```

Comment:
These type based speed mods should all be under 1 entry and handled under the base `update_config_movespeed()`